LCM/scripts/OMS_MetaConfigHelper.py

```diff
@@ -4,6 +4,7 @@
 
 conf_path = "/etc/opt/microsoft/omsagent/conf/omsadmin.conf"
 metamof_path = "/etc/opt/omi/conf/omsconfig/generated_meta_config.mof"
+oms_agentid_path = "/etc/opt/microsoft/omsagent/agentid"
```

Comment:
It is written here: https://github.com/Microsoft/OMS-Agent-for-Linux/pull/527/files#diff-9a868193b8ffdccfa122caa1ff6a6620R187
Yes, this is executed when the user onboards to the first workspace. It will be written before OMS_MetaConfigHelper.py is called.